sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/EventDataExtensionTests.cs

```diff
@@ -76,10 +76,10 @@ public void IsEquivalentToIgnoresSystemPropertiesByDefault()
             var secondEvent = new EventData((byte[])body.Clone());
 
             firstEvent.SystemProperties = new EventData.SystemEventProperties();
-            firstEvent.SystemProperties["something"] = "trackOne";
+            firstEvent.SystemProperties.PartitionKey = "-1";
```

Comment:
I'm genuinely curious about why `trackOne` became `-1`, but no problems here.

Reply:
Got it. They're both Track Two.

Reply:
Yeah, I don't remember making that change, but its likely that I saw the "trackOne" value and it was confusing.   The source was likely a copy/paste from the track one comparer stuff.